models/_sector/dex/trades/arbitrum/platforms/arbswap_arbitrum_base_trades.sql

```diff
@@ -0,0 +1,76 @@
+{{
+    config(
+        schema = 'arbswap_arbitrum',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+WITH dexs AS (
+    -- Arbswap AMM
```

Comment:
this dex is a good example of open questions as we move forward. from what i can tell, the first source query here is [uniswap v2 compatible](https://github.com/duneanalytics/spellbook/blob/main/macros/models/_sector/dex/uniswap_compatible_trades.sql#L1)

i wonder if we explore mixing a combo of macro and union of non-macro code?